snapshot.go

```diff
@@ -9,12 +9,12 @@ package knoxite
 
 import (
 	"encoding/json"
+	uuid "github.com/nu7hatch/gouuid"
```

Comment:
Note: If you keep the empty line between the regular imports and the external ones, go fmt will treat and sort it as a separate block.